fastify.js

```diff
@@ -117,7 +117,7 @@ function build (options) {
     }
   })
 
-  if (Number(process.versions.node[0]) >= 6) {
+  if (Number(process.version.match(/v(\d+)/)[1]) >= 6) {
```

Comment:
Can you write the check here in the same way as below?
`semver.gt(process.versions.node, '6.0.0')`?

Reply:
I could. In that case, `semver` will need to be moved from `devDependencies` to `dependencies`. Is that okay?

Reply:
Uh, you are right!
Let's leave it as is :)